Experience.html

```diff
@@ -98,19 +109,13 @@ <h2 class="UNICEF">Physical Education Teacher (UNICEF, Syria)
                         <li>Coaching social sporting teams
                         </li>
                     </ul>
-        
-
-
-       
-        
-
-
-      
-   
-    
-    
-    
-       
+                  </nav>
+         
+                  <footer>
+                     <p class="copyright"> copyright 2020 reem alawad
+                    </p>
+                   </footer>  
 
     </main>
+   </body>
 </html>
```

Comment:
Also noticed this filename is `Experience.html` e.g. starts with a capital `E` unlike the other html files.

Good practice to keep the file names consistent e.g. `experience.html`